Lab1/Lab1.m

```diff
@@ -0,0 +1,150 @@
+
+%% Section 3A
+% I selected a value of 3.2 as my hypothetical measurement. 
+
+% Section 3B
+% The statistical question I want to ask is "What is the probability
+% of the background producing a signal of 5.4 or larger?"
+
+%% Section 3C
+% If I were to convert my statistical question into an integral, I would
+% need to evaluate the area underneath the distribution from the stated
+% hypothetical value to infinity. These are the lower and upper bounds
+% of the integral respectively. This is equivalent to integrating the 
+% distribution from negative infinity to the hypothetical value and 
+% subtracting one by this result. However, the area underneath the
+% distribution from negative infinity to zero is zero so we are really
+% calculating the area from 0 to z where z is 5.4. Now, we want the
+% probability of producing a signal of 5.4 or larger so we subtract area
+% calculated by the integral of the exponential distribution from negative
+% infinity to 5.4 from one. 
+
+%% Section 3D
+% Calculates the probabilty of background producing a signal.
+expD = makedist("Exponential",'mu',3);
+pTail = 1-cdf(expD,5.4); 
+disp("    "+pTail*100+"%")
+
+%% Section 3E
+% Converts probabilty into equivalent sigma.
+pHead = cdf(expD,5.4);
+disp(norminv(pHead))
+
+%% Section 4
+% Some patterns I noticed from using various hypothetical measurement
+% values is how the probabilities and the sigmas change. Specifically,
+% as the hypothetical measurement increases, the probability decreases
+% while sigma increases. Also, as the hypothetical measurement decreases,
+% the probability increases while sigma decreases. 
+expD = makedist("Exponential",'mu',3);
+pTail = 1-cdf(expD,2.6); 
+disp("    "+pTail*100+"%")
+pHead = cdf(expD,2.6);
+disp(norminv(pHead))
+
+expD = makedist("Exponential",'mu',3);
+pTail = 1-cdf(expD,7.8); 
+disp("    "+pTail*100+"%")
+pHead = cdf(expD,7.8);
+disp(norminv(pHead))
+
+%% Non-Continuous Distributions
+
+%% Section 1B
+% Plots a series of plots with different parameters. One feature I notice
+% is a stair-like plot to prevent noninteger events. For example, it
+% doesn't make sense for 5.4 cellphones to be sold but it makes sense for
+% there to only be 5 or 6 cellphones to be sold. Another feature I notice
+% is the number of instances in x = 0 decrease as lambda increases. To add
+% on to this, there are the most instances lambda. 
+x = 0:10;
+poisson = makedist("Poisson",'lambda',1); 
+subplot(1,2,2)
+stairs(x-0.5,pdf(poisson,x),"LineWidth",2);
+set(gca,'YScale','log')
+subplot(1,2,1)
+stairs(x-0.5,pdf(poisson,x),"LineWidth",2); figure;
+
+x = 0:10;
+poisson = makedist("Poisson",'lambda',6); 
+subplot(1,2,2)
+stairs(x-0.5,pdf(poisson,x),"LineWidth",2);
+set(gca,'YScale','log')
+subplot(1,2,1)
+stairs(x-0.5,pdf(poisson,x),"LineWidth",2); 
+
+%% Section 1C
+% My hypothetical question is "What is the probability of the background
+% producing a signal of 3.2?". Calculates the corresponding probability. 
```

Comment:
What probability did you get for this? I would except it to be zero since 3.2 is probably not one of your allowed values in the discreet probability.

Reply:
![image](https://user-images.githubusercontent.com/72474045/95667186-5e595c00-0b17-11eb-93c9-518f70019ec0.png)

Reply:
I will edit my question to say 3.2 or larger since that is what I am calculating.